raven/src/main/java/org/opencadc/raven/RavenInitAction.java

```diff
@@ -184,18 +199,61 @@ void initStorageSiteRules() {
         log.info("initStorageSiteRules: OK");
     }
 
+    private void initKeyPair() {
+        log.info("initKeyPair: START");
+        jndiPreauthKeys = appName + "-" + PreauthKeyPair.class.getName();
+        try {
+            Map<String,Object> dc = getDaoConfig(props);
+            PreauthKeyPairDAO dao = new PreauthKeyPairDAO();
+            dao.setConfig(dc);
+            PreauthKeyPair keys = dao.get(KEY_PAIR_NAME);
+            if (keys == null) {
+                KeyPair kp = RsaSignatureGenerator.getKeyPair(4096);
```

Comment:
So you are changing the DB account for `raven`?

Reply:
I decided to add a second pool for admin, which also allows raven to init the inventory tables instead of waiting for fenwick to have been run